js-api-spec/legacy/render.test.ts

```diff
@@ -2,9 +2,221 @@
 // MIT-style license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
+import * as p from 'path';
 import * as sass from 'sass';
 
-import {skipForImpl, captureStdioAsync} from '../utils';
+import {sandbox} from '../sandbox';
+import {skipForImpl, captureStdio, captureStdioAsync} from '../utils';
+
+skipForImpl('sass-embedded', () => {
```

Comment:
Does embedded only support the new API, or are we skipping these for some other reason?

Reply:
It currently only supports the `async()` render method, but eventually it will support the full API. I want to land a full test suite that's working with Dart Sass before I start implementing it in the embedded host.